Create ui-c8y-1023-14-27-added-fix-for-data-point-graph-when-you-export-as-image.md (#4072)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-14-27-added-fix-for-data-point-graph-when-you-export-as-image.md

---
date: ""

> title: Fixed display issue when exporting data point graph widget as image.

Reviewer: ```suggestion
title: Data point graph correctly displayed when exported as an image
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65379
version: 1023.14.27
---

> Previously, exporting the data point graph widget as an image could result in the legend overlapping or being displayed incorrectly, making the image hard to read. This has been fixed, and exported images now display the graph correctly without the legend.

Reviewer: ```suggestion
Previously, when exporting the "Data point graph" widget as an image, the legend could overlap or be displayed incorrectly, making the image hard to read. This issue has been fixed, and exported images now display the data point graph correctly.
```

---
date: ""

> title: Added fix for data point graph when you export as image.

Reviewer 2: ```suggestion
title: Fixed display issue when exporting data point graph widget as image.
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65379
version: 1023.14.27
---
# Backport

This will backport the following commits from `develop` to
`release/y2026`:
- [fix(Web SDK): [MTM-65379] Added fix for data point graph when you
export as image.
(#10597)](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/10597)

<!--- Backport version: 9.5.1 -->

### Questions ?
Please refer to the [Backport tool
documentation](https://github.com/sorenlouv/backport)

[MTM-65379]:
https://cumulocity.atlassian.net/browse/MTM-65379?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

> Co-authored-by: Enio Sultan <92023325+eniosultan@users.noreply.github.com>

Reviewer 2: ```suggestion
Previously, exporting the data point graph widget as an image could result in the legend overlapping or being displayed incorrectly, making the image hard to read. This has been fixed, and exported images now display the graph correctly without the legend.
```
